components/provisioner/internal/gardener/auditlogging.go

```diff
@@ -0,0 +1,107 @@
+package gardener
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"regexp"
+
+	gardener_types "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+)
+
+const auditLogConditionType = "AuditlogServiceAvailability"
```

Comment:
Did you look at the Gardener packages for this constant? If there is no way to get it through a package then it's ok, but if we can make it not hardcoded it'd be much better

Reply:
I did, but couldn't find it

Reply:
Oh, ok 😢